Library/Homebrew/os/mac.rb

```diff
@@ -60,6 +60,15 @@ def prerelease?
       version >= "12"
     end
 
+    sig { returns(String) }
+    def preferred_perl_version
+      if version >= :big_sur
+        "5.30"
+      else
+        "5.18"
+      end
```

Comment:
We hacve the option to use 5.28 on Catalina if we want. (5.18 is the default.)

Reply:
@Bo98 May as well?